conflowgen/application/services/inbound_and_outbound_vehicle_capacity.py

```diff
@@ -0,0 +1,165 @@
+from typing import Dict
+
+import numpy as np
+
+from conflowgen.data_summaries.data_summaries_cache import DataSummariesCache
+from conflowgen.descriptive_datatypes import ContainerVolumeByVehicleType
+from conflowgen.domain_models.data_types.mode_of_transport import ModeOfTransport
+from conflowgen.descriptive_datatypes import OutboundUsedAndMaximumCapacity
+from conflowgen.domain_models.distribution_repositories.container_length_distribution_repository import \
+    ContainerLengthDistributionRepository
+from conflowgen.domain_models.distribution_repositories.mode_of_transport_distribution_repository import \
+    ModeOfTransportDistributionRepository
+from conflowgen.domain_models.factories.fleet_factory import create_arrivals_within_time_range
+from conflowgen.domain_models.large_vehicle_schedule import Schedule
+
+
+class InboundAndOutboundVehicleCapacity:
```

Comment:
This sounds like a datatype, not a proper class. Could you add some kind of suffix? In the simplest case, maybe just append `Service`? Or to make it more verbose, we could add `CalculatorService`? Or maybe you have a better idea?

Reply:
See [Rename InboundAndOutboundVehicleCapacity to InboundAndOutboundVehicleCapacityCalculatorService](https://github.com/1kastner/conflowgen/pull/187/commits/27b5c5f392cc91bf6ccc57e17272b5185569da61)